frontend/src/components/ToDoLists.js

```diff
@@ -64,6 +64,16 @@ export const ToDoLists = ({ userState }) => {
             setSelectedList(res)
         }
     }
+    // handler for deleting list
+    const handleDeleteList = async (listId) => {
+        if (!window.confirm("Are you sure?")) {
```

Comment:
We should put a TODO for this to update it to pop up a modal instead of a window confirmation